plugins/catalog-import/src/components/ImportInfoCard/ImportInfoCard.test.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  ApiProvider,
+  ApiRegistry,
+  ConfigReader,
+} from '@backstage/core-app-api';
+import { configApiRef } from '@backstage/core-plugin-api';
+import { wrapInTestApp } from '@backstage/test-utils';
+import { act, render } from '@testing-library/react';
+import React from 'react';
+import { CatalogImportApi, catalogImportApiRef } from '../../api';
+import { ImportInfoCard } from './ImportInfoCard';
+
+describe('<ImportInfoCard />', () => {
+  let apis: ApiRegistry;
+  let catalogImportApi: jest.Mocked<CatalogImportApi>;
+
+  beforeEach(() => {
+    catalogImportApi = {
+      analyzeUrl: jest.fn(),
+      submitPullRequest: jest.fn(),
+    };
+
+    apis = ApiRegistry.with(
+      configApiRef,
+      new ConfigReader({
+        integrations: {
+          github: [{ token: 'my-token' }],
+        },
+      }),
+    ).with(catalogImportApiRef, catalogImportApi);
+  });
+
+  it('renders without exploding', async () => {
+    apis = ApiRegistry.with(
+      configApiRef,
+      new ConfigReader({ integrations: {} }),
+    ).with(catalogImportApiRef, catalogImportApi);
+
+    await act(async () => {
```

Comment:
hint: `await renderInTestApp` is a superb `act` + `wrapInTestApp` + `render`, and can probably be used pretty much always, especially instead of the wrap+render duo. It also returns the outcome, so you could place the `expect` on the outside

Reply:
Just copied that code around, but that pattern is indeed a bit strange. Went through all files to fix it!